lib/gooddata/models/project.rb

```diff
@@ -33,7 +33,8 @@ class Project < GoodData::Rest::Resource
         },
         'content' => {
           'guidedNavigation' => 1,
-          'driver' => 'Pg'
+          'driver' => 'Pg',
+          'environment' => 'PRODUCTION'
```

Comment:
I guess this is a default value which will be used if you use Ruby SDK. But I don't see how you update the tests so that it use TESTING environment. Hm?